src/providers/ror2/downloading/ThunderstoreDownloaderProvider.ts

```diff
@@ -45,7 +45,15 @@ export default abstract class ThunderstoreDownloaderProvider {
                                callback: (progress: number, modName: string, status: number, err: R2Error | null) => void,
                                completedCallback: (modList: ThunderstoreCombo[]) => void): void;
 
-    /**
+
+    public abstract downloadCombos(
```

Comment:
If `combos` passed in and the return value out of the function are actually the same array, then passing it around is IMO detrimental for clarity.

```typescript
// Makes me think :(
const combosToDownload = getDependencies(combo);
const combosToInstall = await download(combosToDownload);
await install(combosToInstall );

// Smooth brain approved ^_^
const combos = getDependencies(combo);
await download(combos);
await install(combos);
```